mypy/test/testcheck.py

```diff
@@ -99,9 +100,6 @@
     'check-formatting.test',
 ]
 
-based_type_check_files = [
-
-]
```

Comment:
this change is also in #139 is that intentional

Reply:
yeah